src/api/app/helpers/webui/obs_factory/application_helper.rb

```diff
@@ -1,5 +1,5 @@
 module Webui::ObsFactory::ApplicationHelper
     def openqa_links_helper
-      OpenqaJob.openqa_links_url
+      ObsFactory::OpenqaJob.openqa_links_url
```

Comment:
This change could be in a8158a0418f208a90227ebe41bdfa7e2f51024ac instead of having its own commit